src/middleware/navigation/remove/remove.journey.middleware.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Middleware to check if this request is for a Remove journey and if so set this as the
+ * journey type (which can then be used by page templates to set the banner etc.).
+ */
+import { NextFunction, Request, Response } from "express";
+
+import {
+  JourneyType
+} from "../../../config";
+import { logger } from "../../../utils/logger";
+import { isRemoveJourney } from "../../../utils/url";
+
+export const removeJourneyMiddleware = (req: Request, res: Response, next: NextFunction) => {
+
+  if (isRemoveJourney(req)) {
```

Comment:
Good idea this makes the remove test more explicit, are there plans to delete the existing isRemoveJourney from url.ts?

Reply:
No, that function will still be needed as called in a number of places (mainly controller code).